easybuild/framework/easyconfig/parser.py

```diff
@@ -79,12 +79,22 @@ class EasyConfigParser(object):
         Can contain references to multiple version and toolchain/toolchain versions
     """
 
-    def __init__(self, filename=None, format_version=None, rawcontent=None):
-        """Initialise the EasyConfigParser class"""
+    def __init__(self, filename=None, format_version=None, rawcontent=None,
+                 auto_convert_value_types=True):
+        """
+        Initialise the EasyConfigParser class
+        @param filename: path to an easyconfig file
+        @param format_version: version of easyconfig file format
+        @param rawcontent: contents of an easyconfig file
```

Comment:
i.e.: raw content of easyconfig file to parse (preferred over easyconfig file supplied via filename!)